_specification-v1/sass.md

```diff
@@ -19,6 +14,11 @@ collection_listing_display: false
 
 Origami component styles are authored in <a href="http://sass-lang.com/" class="o-typography-link--external">Sass</a>, specifically the SCSS syntax. Sass features **should** be used only where they result in increased clarity and reuse. Care should be taken that the resulting CSS is not compromised by unnecessary Sass nesting.
 
+## Sass compiler
+
+Origami components **must** be able to be compiled with the primary implementation of Sass with a <a href="https://sass-lang.com/documentation/at-rules/use#load-paths" class="o-typography-link--external">load path</a> that only contains `node_modules`.
```

Comment:
```suggestion
Origami components **must** be able to be compiled with the <a href="https://sass-lang.com/dart-sass" class="o-typography-link--external">Dart-sass</a> compilter with a <a href="https://sass-lang.com/documentation/at-rules/use#load-paths" class="o-typography-link--external">load path</a> that only contains `node_modules`.
```
Reverting because of the issues [@chee pointed out](https://github.com/Financial-Times/origami-website/pull/273#discussion_r512765591) about components becoming spec noncompliant if a new primary sass implemenation is released and they do not work with it.